manifests/bosh-manifest/spec/manifest_validation_spec.rb

```diff
@@ -86,4 +86,20 @@
       end
     end
   end
+
+  describe "cross-references runtime-config" do
```

Comment:
Test looks good to me

Reply:
When adding keys for Rafal and Miki, I couldn't work out what I'd done wrong, so I modified this test to help me diagnose the problem.